DESIGN.md

```diff
@@ -1,8 +1,9 @@
 # astro
```

Comment:
probably good to note that this is forthcoming, so folks don't get confused.

Reply:
Oh absolutely, that's why we're not putting it in `master`. `main` will be the base branch for v2 so anyone that just visits GitHub won't see this doc